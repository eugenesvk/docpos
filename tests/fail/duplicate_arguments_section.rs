use doxidize::*;

#[doxidize]
/// here are some comments
/// and some more
#[arguments_section]
/// and some more
/// but this next argument section should not be here
#[arguments_section]
pub fn add(
    /// some comments
    first: i32,
    second: i32,
) -> i32 {
    first + second
}

pub fn main() {}
