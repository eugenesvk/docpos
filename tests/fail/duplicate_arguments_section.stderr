error: Duplicate attribute not allowed.
 --> tests/fail/duplicate_arguments_section.rs:9:1
  |
9 | #[parameters_section]
  | ^^^^^^^^^^^^^^^^^^^^^
